test/testTauTriggerSFs.cc

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include "boost/format.hpp"
+
+#include "TString.h"
+#include "TauAnalysisTools/TauTriggerSFs/interface/TauTriggerSFs2017.h"
+
+int main()
+{
+    static const std::vector<std::string> years = { "2016", "2017", "2018" };
+    static const std::vector<std::string> channels = { "ditau", "mutau", "etau" };
+    static const std::vector<std::string> working_points = {
+        "vvvloose", "vvloose", "vloose", "loose", "medium", "tight", "vtight", "vvtight"
+    };
+
+    static const std::vector<double> test_pts = { 45, 80, 120, 400 };
+    static const std::vector<int> decay_modes = { 0, 1, 10, 11 };
+
+    for (std::string year: years)
+    {
+        for (std::string chan: channels)
+        {
+            for (std::string wp: working_points)
+            {
+                TauTriggerSFs2017 sf_reader = TauTriggerSFs2017(chan, year, wp, "DeepTau", true);
```

Comment:
For completeness, I suggest testing the reader for both embedded and MC SFs. Same applies for `testTauTriggerSFs.py`.

Reply:
Suggestion is implemented now.